### PR TITLE
Geometry checker updates

### PR DESCRIPTION
**Pull request by stra2da**
Changes mainly related to:
- new options available in the plugin have been added
-some GUI convetions added
-cut and paste of the first part related to the second tab to have it all toghether after the figure
-figure of the first tab updated (screenshot made with windows, screen color setting: RGB 64,64,64)

### CODE REVIEW

File: source/docs/user_manual/plugins/plugins_geometry_checker.rst [hunk 1]
```diff
@@ -11,38 +11,44 @@ Geometry Checker Plugin
...
-  :guilabel:`Self intersections`, :guilabel:`Duplicate nodes`,
-  :guilabel:`Polygon with less than 3 nodes`.
+validity of a layer. It is available from the :menuselection:`Vector -->`
+|geometryChecker| :menuselection:`Geometry Checker` menu.
+
+The :guilabel:`Geometry Checker` dialog show different grouped settings in the
+first tab (:guilabel:`Setup`):
+
+* :guilabel:`Input vector layers`: to select the layers to check. A |checkbox|
+  :guilabel:`Only selected features` checkbox can filter the geometry to the one selected.
```
Comment (havatv) [inline source/docs/user_manual/plugins/plugins_geometry_checker.rst]: can be used to restrict the checking to the geometries of the selected features.

File: source/docs/user_manual/plugins/plugins_geometry_checker.rst [hunk 2]
```diff
@@ -11,38 +11,44 @@ Geometry Checker Plugin
...
+  |checkbox| :guilabel:`Checks for duplicates`, |checkbox| :guilabel:`Checks
+  for features within other features`, |checkbox| :guilabel:`Checks for
+  overlaps smaller than` |selectNumber|, |checkbox| :guilabel:`Checks
+  for gaps smaller than` |selectNumber|, |checkbox| :guilabel:`Points
+  must be covered by lines`, |checkbox| :guilabel:`Points must properly lie
+  inside a polygon`, |checkbox| :guilabel:`Lines must not intersect any other
+  lines`, |checkbox| :guilabel:`Lines must not intersect with features of
+  layer` |selectString|, |checkbox| :guilabel:`Polygons must follow boundaries
+  of layer` |selectString|.
 * :guilabel:`Tolerance`: you can define here the tolerance for the check.
```
Comment (havatv) [inline source/docs/user_manual/plugins/plugins_geometry_checker.rst]: How about: "define the tolerance of the check in map layer units."

File: source/docs/user_manual/plugins/plugins_geometry_checker.rst [hunk 3]
```diff
@@ -11,38 +11,44 @@ Geometry Checker Plugin
...
 * :guilabel:`Output vector layer` gives the choice to the user how get the
   result between modify the current layer and create a new layer.
 
-After you are happy with the configuration, you can click on the **[Run]** button.
+After you are happy with the configuration, you can click on the **[Run]**
```
Comment (havatv) [inline source/docs/user_manual/plugins/plugins_geometry_checker.rst]: When you are ...

File: source/docs/user_manual/plugins/plugins_geometry_checker.rst [hunk 4]
```diff
@@ -73,23 +80,35 @@ The following figure shows the different checks made by the plugin.
...
+   Some checks supported by the plugin
 
-You can select a row to see the localisation of the error. You can change this
-behaviour by selecting another action between :guilabel:`error` (default),
-:guilabel:`Feature`, :guilabel:`Don't move`, and |checkbox| `Highlight contour
-of selected features`.
+The results appear in the second tab (:guilabel:`Result`) and as an overview
+layer of the errors in the canvas (its name has the default prefix
+:file:`checked_`).
+A table list the :guilabel:`Geometry check result` with one error by row and
```
Comment (havatv) [inline source/docs/user_manual/plugins/plugins_geometry_checker.rst]: "A table lists" ("lists", not "list")
"one error per row"

File: source/docs/user_manual/plugins/plugins_geometry_checker.rst [hunk 5]
```diff
@@ -73,23 +80,35 @@ The following figure shows the different checks made by the plugin.
...
+resolution column which indicates the resolution of the error.
+At the bottom of this table, you can **[Export]** the error into different file
+formats. You also have a counter with the number of total errors and fixed ones.
 
-Below the zoom action when clicking on the table row, you can :guilabel:`Show
-the selected features in attribute table`, :guilabel:`Fix selected errors using
-default resolution` and :guilabel:`Fix selected errors, prompt for resolution
-method`. In the latter, you will see a window to choose the resolution's method
-among which:
+You can select a row to see the localisation of the error. You can change this
```
Comment (havatv) [inline source/docs/user_manual/plugins/plugins_geometry_checker.rst]: location ?

**Comment by stra2da** [comment]
Thanks for your suggestions! I'm not English mother-tongue so I've tried to change only the minimum required. (And sorry for the chaotic commit history in the pull request :) )

**Comment by havatv** [comment]
Thanks, @stra2da!

**Pull request merged**